website/content/api-docs/secret/pki.mdx

```diff
@@ -799,544 +808,504 @@ $ curl \
-  disabled, as large amount of leases adversely affect the startup time of Vault.
+#### Sample Request
 
-- `no_store` `(bool: false)` – If set, certificates issued/signed against this
-  role will not be stored in the storage backend. This can improve performance
-  when issuing large numbers of certificates. However, certificates issued in
-  this way cannot be enumerated or revoked, so this option is recommended only
-  for certificates that are non-sensitive, or extremely short-lived. This
-  option implies a value of `false` for `generate_lease`.
+```shell-session
+$ curl \
+    http://127.0.0.1:8200/v1/pki/issuer/root-x1/crl
+```
 
-- `require_cn` `(bool: true)` - If set to false, makes the `common_name` field
-  optional while generating a certificate.
+#### Sample Response
 
-- `policy_identifiers` `(list: [])` – A comma-separated string or list of policy
-  OIDs.
+```json
+{
+  "data": {
+    "crl": "-----BEGIN X509 CRL-----\nMIIBizB1AgEBMA0GCSqGSIb3DQEBCwUAMBIxEDAOBgNVBAMTB3Jvb3QgeDEXDTIy\n..."
+  }
+}
+```
 
-- `basic_constraints_valid_for_non_ca` `(bool: false)` - Mark Basic Constraints
-  valid when issuing non-CA certificates.
+### List Certificates
+
+This endpoint returns a list of the current certificates by serial number only.
+The response does not include the [special serial numbers](#read-certificate-serial-param-values)
+(`ca`, `ca_chain`, and `crl`) that can be used with `/pki/cert/:serial`.
+
+~> Note: The endpoint to list all certificates is authenticated. This is to
+   prevent automated enumeration of issued certificates for internal services;
+   however, this information should generally be considered non-sensitive and
+   the certificates themselves are exposed without authentication (provided
+   their serial number is known).<br /><br />
+   Many Public CAs participate in the Certificate Transparency initiative,
+   where all issued certificates are publicly disclosed in the interest
+   of third-party verification of CA integrity.
+
+| Method | Path         |
+| :----- | :----------- |
+| `LIST` | `/pki/certs` |
 
-- `not_before_duration` `(duration: "30s")` – Specifies the duration by which to backdate the NotBefore property.
+#### Sample Request
 
-- `not_after` `(string)` – Set the Not After field of the certificate with
-  specified date value. The value format should be given in UTC format
-  `YYYY-MM-ddTHH:MM:SSZ`. Supports the Y10K end date for IEEE 802.1AR-2018
-  standard devices, `9999-12-31T23:59:59Z`.
+```shell-session
+$ curl \
+    --header "X-Vault-Token: ..." \
+    --request LIST \
+    http://127.0.0.1:8200/v1/pki/certs
+```
 
-### Sample Payload
+#### Sample Response
 
 ```json
 {
-  "allowed_domains": ["example.com"],
-  "allow_subdomains": true
+  "data": {
+    "keys": [
+      "17:67:16:b0:b9:45:58:c0:3a:29:e3:cb:d6:98:33:7a:a6:3b:66:c1",
+      "26:0f:76:93:73:cb:3f:a0:7a:ff:97:85:42:48:3a:aa:e5:96:03:21"
+    ]
+  }
 }
 ```
 
-### Sample Request
+<a name="read-raw-certificate"></a>
```

Comment:
Is this an anchor?  You can automatically refer to sections by name by converting them to lowercase snake casing.  Eg read_certificate.

Reply:
Yes, but this is an anchor leftover from previous versions of this document, manually re-introduced so bookmarks don't break. See also: https://github.com/hashicorp/vault/pull/15238#discussion_r866263389